playground/src/server/index.ts

```diff
@@ -127,7 +142,8 @@ export async function spawnSxgServer({
     try {
       sxgId = await createSxgIntoList(
         sxgInnerUrl,
-        `https://${request.hostname}`
+        `https://${request.hostname}`,
+        /*subresourceLevel=*/ 1
```

Comment:
Consider making this just `/*isTopLevel=*/ true`. I don't think there are any plans to support multiple levels of prefetch recursion.

Reply:
Done